src/elements/fields/index.jsx

```diff
@@ -189,6 +191,37 @@ function applyFieldStyles(field, styles) {
       styles.applyCorners('field');
       styles.applyBorders('field');
       break;
+    case 'payment_method':
+      styles.addTargets('sub-fc', 'tooltipIcon', 'completed');
+      styles.applyHeight('sub-fc');
+      // styles.applyWidth('sub-fc');
```

Comment:
double check if you need this style or not

Reply:
Not needed, width is in 'fc' only.